Source/WebCore/platform/graphics/PathImpl.h

```diff
@@ -67,7 +67,7 @@ class PathImpl : public ThreadSafeRefCounted<PathImpl> {
     void addLinesForRect(const FloatRect&);
     void addBeziersForRoundedRect(const FloatRoundedRect&);
 
-    virtual void applySegments(const PathSegmentApplier&) const = 0;
+    virtual void applySegments(const PathSegmentApplier&) const;
```

Comment:
I guess this doesn't need to be virtual anymore.

Reply:
Well, there's one more override left in the codebase - `PathStream::applySegments()`. The thing with this one is - it's implementation is more liberate than of `PathStream::applyElements()` that does some extra filtering. Therefore removing `PathStream::applySegments()` would change behavior of applying.